packages/dd-trace/src/appsec/sdk/utils.js

```diff
@@ -1,8 +1,26 @@
 'use strict'
 
 function getRootSpan (tracer) {
```

Comment:
we get the rootspan everyone in appsec's code, is this the only place that needs to be updated ?

Reply:
Yes, the problem is only with the SDK.